uid/polymorphic.go

```diff
@@ -35,6 +35,10 @@ func NewGroupPolymorphicID(id ID) PolymorphicID {
 	return newPolymorphicID("g", id)
 }
 
+func NewOrganizationPolymorphicID(id ID) PolymorphicID {
```

Comment:
Do we need polymorphic IDs for orgs? I think these are only used in grants. Not sure if we want to do org-wide grants.